docs/pages/api-reference/client.md

```diff
@@ -8,14 +8,14 @@ status: WIP
 
 Fennel Client has the following methods on it:
 
-### extract\_features
+### extract
 
 Given some input and output features, extracts the current value of all the output features given the values of the input features.
 
 **Arguments:**
 
-* `output_feature_list: List[Union[Feature, Featureset]]`: list of features (written as fully qualified name of a feature along with the featureset) that should be extracted. Can also take featureset objects as input, in which case all features in the featureset are extracted.
-* `input_feature_list: List[Union[Feature, Featureset]]` : list of features/featuresets for which values are known
+* `outputs: List[Union[Feature, Featureset]]`: list of features (written as fully qualified name of a feature along with the featureset) that should be extracted. Can also take featureset objects as input, in which case all features in the featureset are extracted.
+* `inputs: List[Union[Feature, Featureset]]` : list of features/featuresets for which values are known
```

Comment:
We should remove Featureset from the type for inputs, since we require explicitly features to be passed. 

We could add an info on why we do so. 

It is because if an engineer adds a new feature to the featureset it would break all extract calls running in production

Reply:
We should add str too and specify in the docs that either the fully qualified feature names can be used  ( when providing a str ) or the feature object

Reply:
done. Make sense to add a line on why we don't allow adding Featureset to input